### PR TITLE
#165413072 Adds like and dislike functionality

### PR DESCRIPTION
**Pull request by Niyitangasam**
Description
=======

This PR is for adding functionalities of liking and disliking an article.

Type of change
=======
N/A

How has it been tested
=======
It can be tested by log in and access below  routes:

- You can access specific article on ``/articles/slug``.

Checklist:
=======
- Add components for liking and disliking an article.
- Add its actions and reducers
- Add necessary HTML and scss to the page
Pivotal tracker story ID
=======
[#165413072](https://www.pivotaltracker.com/n/projects/2326984/stories/165413072)

Screenshots
=======
image

### CODE REVIEW

File: src/components/common/navBar.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            <a href='#'>New story</a>
+            <a href='./index.html'>Sign out</a>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+ export default NavBar;
```
Comment (jabichris) [inline src/components/common/navBar.jsx]: Can you add a new line here?

**Review by Kabalisa — CHANGES_REQUESTED** [review]
@Niyitangasam Good work.  the feature is working fine. however, the design on the like and dislike icon is different from the one implemented in our templates. you can pick up the sass we used to change it.

**Review by Kabalisa — APPROVED** [review]
good work @Niyitangasam .

File: src/assets/scss/_oneStory.scss [hunk 2]
```diff
@@ -1,126 +1,140 @@
...
-  background-color: #c54848;
-  color: #ffffff;
-  width: 60px;
-  font-size: 12px;
-  padding: 10px;
-  border: 0px;
-  border-radius: 5px;
-  margin-right: 10px;
-}
+
```
Comment (jabichris) [inline src/assets/scss/_oneStory.scss]: Remove a new line here

**Review by jabichris — APPROVED** [review]
Nice work

**Pull request merged**